src/fake-timers-src.js

```diff
@@ -1848,6 +1848,8 @@ function withGlobal(_global) {
                     new FakePerformanceEntry(name, "mark", 0, 0);
                 clock.performance.measure = (name) =>
                     new FakePerformanceEntry(name, "measure", 0, 100);
+                // ensure `timeOrigin` returns a valid value
+                clock.performance.timeOrigin = 0;
```

Comment:
> If the current [Document](https://developer.mozilla.org/en-US/docs/Web/API/Document) is the first one loaded in the Window, the time origin is the time at which the browser context was created.

Best fit is likely the installed time

Reply:
Would this work?

```js
clock.performance.timeOrigin = config.now ? getEpoch(config.now) : new Date().getTime();
```

Reply:
A bit redundant, given that `getEpoch` already does just that. I trimmed it away and also made the test a bit tighter/more specific.

Reply:
Thank you!